### PR TITLE
Remove uses of classNames that can be plain strings 

### PR DESCRIPTION
**Pull request by alexwlchan**
We only need to use classNames when there's a condition that might mean a style isn't being used.  If all the conditions are `true`, we can remove the function and replace it with a string interpolation.

Note: in a few cases, I've made what might seem like non-obvious choices for formatting the string, e.g. in Quote.tsx:

```typescript
className={
  'quote__cite flex flex--v-end font-pewter' + ' ' + font('intr', 5)
}
```

rather than

```typescript
className={
  `quote__cite flex flex--v-end font-pewter ${font('intr', 5)}`
}
```

This is me fighting the auto-formatter and trying to preserve readability; in that example, the code gets formatted as

```typescript
className={
  `quote__cite flex flex--v-end font-pewter ${font(
    'intr',
     5
  )}`
}
```

which seems far less readable to me.  I've tried to keep small functions like `font()` and `grid()` on a single line, even if it means making the overall className look a bit different.

For #8419

### CODE REVIEW

File: common/views/components/LinkLabels/LinkLabels.tsx [hunk 1]
```diff
@@ -85,22 +69,14 @@ const LinkLabels: FunctionComponent<Props> = ({
...
-        flex: true,
-        'plain-list': true,
-        'flex--wrap': true,
-        'no-margin': true,
-        'no-padding': true,
-        [font('intb', 5)]: true,
-      })}
+      className={
+        'flex plain-list flex--wrap no-margin no-padding' +
+        ' ' +
```
Comment (davidpmccormick) [inline common/views/components/LinkLabels/LinkLabels.tsx:74]: Could use string interpolation here too

File: content/webapp/pages/whats-on.tsx [hunk 2]
```diff
@@ -548,17 +530,13 @@ const WhatsOnPage: FunctionComponent<Props> = props => {
...
+              <div className="css-grid">
                 <div
-                  className={classNames({
-                    'css-grid__scroll-container container--scroll touch-scroll':
-                      true,
-                    [cssGrid({ s: 12, m: 12, l: 12, xl: 12 })]: true,
-                  })}
+                  className={
+                    'css-grid__scroll-container container--scroll touch-scroll' +
+                    ' ' +
```
Comment (davidpmccormick) [inline content/webapp/pages/whats-on.tsx:537]: Interpolation?

**Review by davidpmccormick — APPROVED** [review]
A handful of instances where (I think) string interpolation would be neater, but maybe not (and obviously not blocking).

**Comment by alexwlchan** [comment]
> A handful of instances where (I think) string interpolation would be neater

In all these cases, using string interpolation would cause a `font(…)` or `grid(…)` declaration to be split across multiple lines by the VSCode autoformatting (see PR description), so I decided against it.

**Pull request merged**